step-framework-model/src/main/java/step/core/collections/Filter.java

```diff
@@ -42,5 +42,7 @@
 })
 public interface Filter {
 
+    String getField();

```

Comment:
@david-stephan the fields from the filters have to be serialized (they have been before). If I annotate this method, there will be no fields serialized anymore. That's because I reused the getField() method which was existing in the entities already. A better/nicer option would be to ingore it when it's null, but I'm not sure if I can do it only for this specific attribute

Reply:
I missed that. But the getField only exists in a subset of the implementing classes where it made sense, so either we have a cleaner implementation that does not reuse that method, or we should be able to at least use following annotation (to be tested) instead of JsonIgnore: @JsonInclude(JsonInclude.Include.NON_NULL)